modules/codegen/src/smithy4s/codegen/ModelLoader.scala

```diff
@@ -59,6 +59,7 @@ object ModelLoader {
       val upstreamModel = Model
         .assembler()
         .discoverModels(upstreamClassLoader)
+        .addClasspathModels(currentClassLoader, false)
```

Comment:
is the comment on line 57 still relevant?

basically, we load to models: 
- one with upstream deps + smithy4s (via current class loader)
- one with everything on the classpath (with discovery enabled)

Reply:
you could be cleaned up

Reply:
Sounds like a threat lmao

Reply:
😆 I meant 'yeah, could be cleaned up'